install_build_dependencies.sh

```diff
@@ -28,6 +28,20 @@ CLANG_VERSION=3.8.0
 echo "Platform: ${PLATFORM}"
 
 case "${PLATFORM}" in
+arch)
+  # If `gcc-multilib` is already installed, don't try to install plain `gcc`
+  pacman -Qqs gcc-multilib > /dev/null
+  if [ $? ]; then
+    sudo pacman --needed -S cmake git python readline
+  else
+    sudo pacman --needed -S cmake git python readline gcc
+  fi
+  # We also need termcap from AUR
+  pacman -Qs termcap
+  if [ $? = 1 ]; then
+    printf "\n!!! You must install termcap from AUR using your preferred tool !!!\n"
```

Comment:
Thank you for the pull request. Messages asking the user to install further components outside of `install_build_dependencies.sh` should be avoided. Could this be automated somehow?

Reply:
Well, yes & no ;-)
The problem is that `termcap` isn't in the official repositories; but there is a recipe provided by a user in AUR. [From the ArchLinux Wiki:] "AUR is a community-driven repository[...] It contains package descriptions that allow you to compile a package from source[...]".

The only official process to create a package from AUR is a manual one; where one downloads a script then creates the package and installs it. So, I could try to automate that process in `install_build_dependencies.sh`. But:
- there is a technical risk that the URL to the recipe changes;
- there is a potential security risk because that script isn't officially validated and tracked. So, as an Arch user, I prefer to be offered the choice to install or not a package from AUR...

To be complete, there are some helpers (https://wiki.archlinux.org/index.php/AUR_helpers) that can be used to improve the user-experience. But none are official, and the most well-known (yaourt) is quite old.

So, if you really prefer, I can try to come with a manual download & build. But in all cases, I think this is better etiquette to ask the user whether he accepts the AUR package...

M.

Reply:
BTW,  kudos for metashell! This is a wonderful tool :-)

Reply:
Please take a look at the `termcap` branch in my fork of this repository (https://github.com/sabel83/metashell/tree/termcap). I have added Termcap's source code to the repository, so Metashell builds Termcap for itself and does not depend on its availability. Let me know if this works for you.

Reply:
So I checked-out the `termcap` branch and removed the AUR termcap package.
Then I erased the `bin` folder and re-applied the commands from `build.sh` (i.e. I didn't want to recompile `templight` and AFAIK it doesn't use termcap).

This produced a new binary that works ok. So I believe everything is fine!

Reply:
I have merged those changes to `master`. Please update your pull request and remove the request to manually install `termcap`.